bulletroboy/test_publisher.py

```diff
@@ -24,24 +25,33 @@ def main():
         # angle.data = 58.0
         # angle_publisher.publish(angle)
 
-        state = CageState()
-        state.rotation_angle = 47.8
-        for i in range(3):
-            muscleunit = MuscleUnit()
-            muscleunit.id = i
-            for i in range(2):
-                point = ViaPoint()
-                point.id = i
-                point.position = Point()
-                point.position.x = 0.0
-                point.position.y = 0.0
-                point.position.z = 0.0
-                point.reference_frame = 'world'
-                point.link = 'hand'
-                muscleunit.viapoints.append(point)
-            state.muscleunits.append(muscleunit)
-
-        cage_state_publisher.publish(state)
+        # state = CageState()
```

Comment:
why would you remove another person's test publisher?

Reply:
just commented it out, it is a test publisher in the end
I can put it back if you think it's crucial

Reply:
i understand that. but why not create a function for your own test stuff?